css-canvas-medical/index.html

```diff
@@ -0,0 +1,151 @@
+    <div class="row gray-background"> <!--Clinicians prefer canvas row-->
+      <div class="column-full margin-top-bottom">
+        <div class="row gray-font justify-center prefer-margin">
+          <h1>Clinicians Prefer Canvas</h1>
+        </div>
+        <div class="row gray-font text-center justify-center note-width margin-auto">
+          <p class="p-font-size">
+            Physician-owned primary care practices across the country have
+            already made Canvas the fabric of their care.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="row gray-background"> <!--Locations Row-->
+      <div class="column-location">
+        <div class="row locations-margin">
+          <div class="column-half">
+            <div class="row gray-font locations justify-center">
+              <p>Cascade Family Practice, Portland OR</p>
+            </div>
+            <div class="row gray-font locations justify-center">
+              <p>Temescal Creek Medicine, Oakland CA</p>
+            </div>
+            <div class="row gray-font locations justify-center">
+              <p>San Juan Healthcare, Friday Harbor WA</p>
+            </div>
+          </div>
+          <div class="column-half">
+            <div class="row gray-font locations justify-center">
+              <p>Ward Medical Clinic, Bonifay FL</p>
+            </div>
+            <div class="row gray-font locations justify-center">
+              <p>Caribou Medical, Towanda PA</p>
+            </div>
+            <div class="row gray-font locations justify-center">
+              <p>Bay Tree Family Care, Red Bay AL</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <hr class="hr-line margin-zero">
+    <div class="row margin-top-bottom"> <!--Ready to learn more row-->
+      <div class="column-one-third padding-left">
+        <div class="row gray-font">
+          <h1>Ready to learn more?</h1>
+        </div>
+        <div class="row gray-font">
+          <p class="margin-zero p-font-size">Schedule a private meeting now</p>
+        </div>
+      </div>
+      <div class="column-one-third"></div>
+      <div class="column-one-third margin-top-auto">
+        <div class="row justify-center">
+          <button class="submit button-started">Get Started</button>
+        </div>
+      </div>
+    </div>
+    <div class="row dark-gray-background link-padding"> <!--Info Row-->
+      <div class="column-one-fourth address-detail padding-left">
+        <div class="row company-product-detail text-body address-detail">
+          <img src="images/logo-white.png" alt="White Canvas Logo" class="white-logo">
+          <p class="margin-zero">2307 Irving Street, Suite 228</p>
+          <p class="margin-zero">San Fransisco, CA 94122</p>
+          <p class="margin-zero">(800) 370-1416</p>
+          <p class="margin-zero">info@canvasmedical.com</p>
+        </div>
+      </div>
+      <div class="column-one-fourth"></div>
```

Comment:
😬 😬 😬 😬 

Remember not to use empty elements to create space. This is likely a job for `padding`, or you can just make the previous element `column-half` instead of `column-fourth`

Reply:
I ended up doing column-half and then column-half. is that ok?

Reply:
Actually, I realized that was unnecessary, too. Changed it so that it's just column-half!